loopout.go

```diff
@@ -504,31 +533,64 @@ func (s *loopOutSwap) payInvoices(ctx context.Context) {
 	)
 }
 
+// paymentResult contains the response for a failed or settled payment, and
+// any errors that occurred if the payment unexpectedly failed.
+type paymentResult struct {
+	status *lndclient.PaymentStatus
```

Comment:
Since we have the `Payment_UNKNOWN` state for value `0`, should we make this an explicit value instead of a pointer?

Reply:
sgtm!